macros/get_lookback_date_limits.sql

```diff
@@ -7,8 +7,14 @@
   -- check if web data is up-to-date
 
   {% set query %}
-    select max(start_tstamp) < '{{ var('snowplow__conversion_window_end_date') }}' as is_over_limit,
+
+      {% if var("snowplow__conversion_window_start_date") == '' %}
+           select false as is_over_limit,
```

Comment:
it could still be over the limit right? Because we filter to yesterday and they might have some records of today?

Reply:
It could actually be, but in case the web model has not run to process everything until yesterday. Will send the changes in a sec.